PowerEditor/src/WinControls/Preference/preferenceDlg.cpp

```diff
@@ -169,7 +169,7 @@ INT_PTR CALLBACK PreferenceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 			_wVector.push_back(DlgInfo(&_searchingSubDlg, TEXT("Searching"), TEXT("Searching")));
 			_wVector.push_back(DlgInfo(&_backupSubDlg, TEXT("Backup"), TEXT("Backup")));
 			_wVector.push_back(DlgInfo(&_autoCompletionSubDlg, TEXT("Auto-Completion"), TEXT("AutoCompletion")));
-			_wVector.push_back(DlgInfo(&_multiInstanceSubDlg, TEXT("Multi-Instance"), TEXT("MultiInstance")));
+			_wVector.push_back(DlgInfo(&_multiInstanceSubDlg, TEXT("Multi-Instance & Date"), TEXT("MultiInstance")));

```

Comment:
How about to put it to "Edit" or "Misc." category?
It looks like it doesn't belong to "Multi-instance".

Reply:
There's no more room in both "Edit" & "Misc." categories.
"Multi-instance" is one of rare pages which have still some room.

Reply:
How about adding a new page?

Reply:
I'm trying not overload the left menu, so it's not considerable for this category.